policy/provider.yaml

```diff
@@ -3,7 +3,5 @@ kind: Provider
 metadata:
   name: tagtodigest-provider
 spec:
-  proxyURL: http://tagtodigest-provider.default:8090/mutate
+  url: http://tagtodigest-provider.default:8090/mutate
   timeout: 3
```

Comment:
since mwh timeout is 2 seconds now, should we decrease this?

Reply:
I still see the mutating webhook timeout being [3 seconds](https://github.com/open-policy-agent/gatekeeper/blob/master/manifest_staging/charts/gatekeeper/values.yaml#L19)

Reply:
chart is updated now, let's update this to 1